### ISSUE TITLE
Unable to access admin after fresh install

### ISSUE THREAD

**Issue by werdlv** [ionize/ionize#381]
**Steps to reproduce**:

1. Install Ionize 1.0.8.1 setting admin user with username `admin` and password `admin`
2. Go to `/admin` and try to login

**Expected**:
User is authenticated

**Actual**:
Authentication error: `The login information you provided could not be authenticated. Either the username or the password you entered are wrong. Please try again.`

Going back to `install/?step=user` and setting the same user once again fixes the problem.

**Comment by partikule** [comment]
Strange.
Can you try with a fresh install of the master from github ?

**Comment by werdlv** [comment]
Tried fresh install from `master` branch - issues is present

**Comment by partikule** [comment]
How long is your user name ?
Did you tried login with the user's email ?

**Comment by werdlv** [comment]
As said - username & password both are `admin`
No luck with user's e-mail either

**Comment by koencaerels** [comment]
Did you had a look at this issue?  "Different encryption in installation and login action afterwards on the admin." With some PHP versions there is a difference in encryption and decryption of pwd's. >  #356

**Comment by werdlv** [comment]
@partikule any update on this?

**Comment by quanvn** [comment]
Yes, I confirm I got the same problem (on a shared hosting service running PHP 5.5, Ionize version 1.0.8.1).  Going back to install/?step=user and setting the same user once again as mentioned by werdlv did fix the problem.

**Comment by AkselMeola** [comment]
Confirmed, had the same issue on 1.0.8.1 (php 5.5.36). Solved by redoing the user step.

**Comment by adaliszk** [comment]
The problem is in the installation script for the first time the application/config/config.php encryption_key is empty and if on the first user creation you dont copy-paste the encryption_key from the installer before you hit the next button the user password will be use different (or empty) encryption key.

For the second run the installer detecting the encryption_key from the config and not generating an other, that why it's working for the second time. Sadly I dont have time to debug it where goes wrong the Installation scrip, but:

I have experienced with ^1.0.5 the issue but with the same version I have installed without the bug on servers, it can be a php-mcrypt related because the Codeigniter when the php-mcrypt available then it trying use that if not then the script are using internal code for the job.
